assignment.js

```diff
@@ -104,10 +119,25 @@ Use the following test cases to confirm your program meets the success criteria
 console.log('Problem 3:')
 
 // Add your code below this line
+for (let i=0; i < 101; i++){
+  if (i % 15 === 0)
```

Comment:
🛠 Read the comment below and update your code accordingly

@jimernest90 best practice is to use the following syntax for if..else if statements

`if (condition) {
 
} esle if { 

} else { 

}
```

Looks like you are missing some curly braces. Please add them